templates/broadcom_icos_show_version.textfsm

```diff
@@ -0,0 +1,41 @@
+Value System_Description (.+)

```

Comment:
Can you please make all the values upper-case?

```
Value SYSTEM_DESCRIPTION (.+)
```

Reply:
Corrected all named values to upper-case